app/scripts/directives/seekBar.js

```diff
@@ -9,53 +9,53 @@
       //Walkthrough this code
       return offsetXPercent;
     };
-    
+
```

Comment:
Lots of whitespace cleanup that my editor does automatically, sorry for the noise.